Configurando vuex

matheussbernardo opened the pull request:
Adicionando vuex ao front end do falko para gerir estado da aplicação.

matheussbernardo committed:
adding vuex for authentication

matheussbernardo committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into vuex_auth

matheussbernardo committed:
fixing state for auth

matheussbernardo committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into vuex_auth

matheussbernardo committed:
improving vuex state

matheussbernardo committed:
Using state from vuex on components

matheussbernardo committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into vuex_auth

matheussbernardo committed:
fixing router with vue persist

matheussbernardo committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into vuex_auth

alaxalves commented:
Ow Zé, seus testes tão quebrando! @matheussbernardo

alaxalves commented:
![image](https://user-images.githubusercontent.com/19597045/32407655-4706fc92-c173-11e7-9ace-73cd7e169424.png)

alaxalves commented:
@matheussbernardo Outra coisa, acabei de aceitar o PR que muda a estrutura de pastas do projeto, só atualize a sua branch com a _devel_ e atualize o PR ;)

cantuariavc commented:
Estou arrumando os conflitos.

cantuariavc committed:
Merging branch 'devel' into branch 'vuex_auth'

Conflicts:
	package-lock.json
	src/App.vue
	src/components/AddProject.vue
	src/components/Authentication/Login.vue
	src/components/Authentication/Register.vue
	src/components/DeleteProject.vue
	src/components/DeleteUserProfile.vue
	src/components/EditProject.vue
	src/components/EditUserProfile.vue
	src/components/GitHub/GitHubCallBack.vue
	src/components/GitHub/GitHubProjects.vue
	src/components/GitHubCallBack.vue
	src/components/GitHubProjects.vue
	src/components/Login.vue
	src/components/Navbar.vue
	src/components/Project.vue
	src/components/Projects.vue
	src/components/Projects/AddProject.vue
	src/components/Projects/DeleteProject.vue
	src/components/Projects/EditProject.vue
	src/components/Projects/Project.vue
	src/components/Projects/Projects.vue
	src/components/Register.vue
	src/components/Releases/AddRelease.vue
	src/components/Releases/DeleteRelease.vue
	src/components/Releases/EditRelease.vue
	src/components/Releases/Release.vue
	src/components/Releases/Releases.vue
	src/components/ReleasesComponents/AddRelease.vue
	src/components/ReleasesComponents/DeleteRelease.vue
	src/components/ReleasesComponents/EditRelease.vue
	src/components/ReleasesComponents/Release.vue
	src/components/ReleasesComponents/Releases.vue
	src/components/Sprints/AddSprint.vue
	src/components/Sprints/DeleteSprint.vue
	src/components/Sprints/EditSprint.vue
	src/components/Sprints/Sprint.vue
	src/components/Sprints/Sprints.vue
	src/components/SprintsComponents/AddSprint.vue
	src/components/SprintsComponents/DeleteSprint.vue
	src/components/SprintsComponents/EditSprint.vue
	src/components/SprintsComponents/Sprint.vue
	src/components/SprintsComponents/Sprints.vue
	src/components/UserProfile.vue
	src/components/Users/DeleteUserProfile.vue
	src/components/Users/EditUserProfile.vue
	src/components/Users/UserProfile.vue

cantuariavc committed:
Fixing Project route

cantuariavc commented:
Testei tudo e a única coisa que eu vi que tem q arrumar é o botão de adicionar retrospectiva.

cantuariavc committed:
Fixing issues of code climate

cantuariavc commented:
@matheussbernardo, veja se você consegue resolver essa issue do code climate e os testes.

cantuariavc committed:
Removing localstorages

cantuariavc committed:
Fixing Retrospective

Signed-off-by: Matheus S Bernardo <matheussbernardo@gmail.com>

matheussbernardo committed:
Fixing test to vuex

Signed-off-by: Vinícius Cantuária <cantuariavc@gmail.com>

cantuariavc committed:
Fixing GitHub tests

cantuariavc committed:
Merging branch 'devel' into branch 'vuex_auth'

Conflicts:
	src/App.vue
	src/components/GitHub/GitHubProjects.vue
	src/components/HomePage.vue
	src/components/Projects/Projects.vue
	src/components/Retrospective/AddRetrospective.vue
	src/components/Sidebar.vue
	src/router/index.js

cantuariavc committed:
Fixing GitHub tests and code climate issue

cantuariavc commented:
![captura de tela 2017-11-12 as 21 06 57](https://user-images.githubusercontent.com/21143590/32704522-abd524f2-c7ed-11e7-9441-deb637e5f271.png)

matheussbernardo committed:
Merge branch 'devel' of https://github.com/fga-gpp-mds/Falko-2017.2-FrontEnd into vuex_auth

matheussbernardo committed:
fixing codeclimate

MatheusRich committed:
Removing localStorage references on SearchBar and Gpa components

Signed-off-by: MateusO97 <mat97_@hotmail.com>
Signed-off-by: cantuariavc <cantuariavc@gmail.com>

MateusO97 committed:
Removing localStorage reference on sidebar

Signed-off-by: cantuariavc <cantuariavc@gmail.com>
Signed-off-by: MatheusRich <matheusrichardt@gmail.com>

MatheusRich commented:
@matheussbernardo Arrumamos a sidebar/navbar. Haviam algumas referências à localstorage ainda.

MatheusRich committed:
Fixing issues component

Signed-off-by: MateusO97 <mat97_@hotmail.com>
Signed-off-by: cantuariavc <cantuariavc@gmail.com>

cantuariavc commented:
Resolve a issue #178
